internal/confmapprovider/discovery/bundle/bundle.d/receivers/redis.discovery.yaml

```diff
@@ -0,0 +1,73 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+redis:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)redis"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)redis" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)redis"
+  config:
+    default:
+      endpoint: splunk.discovery.default
```

Comment:
This will be set from the discovered endpoint so I'm not sure we should be setting a default.